documentation/assemblies/assembly-iot-tenant-guide.adoc

```diff
@@ -7,49 +7,23 @@
 [id='iot-tenant-{context}]
 = IoT for messaging tenants
 
+Messaging tenants are typically responsible for sending telemetry and events using IoT supported protocols.
```

Comment:
I would suggest to use something more broad here (as we'll add more material in the future).

"Messaging tenants are typically responsible for managing devices and writing device and back-end applications of IoT solutions"